### PR TITLE
[release-v1.11] fix: triggers use correct consumer group id generated from template (#3779)

### PR DESCRIPTION
**Pull request by Cali0707**
Backport of https://github.com/knative-extensions/eventing-kafka-broker/pull/3779

### CODE REVIEW

**Comment by Cali0707** [comment]
@matzew @pierDipi I'll update this PR to include https://github.com/knative-extensions/eventing-kafka-broker/pull/3782 as well once it merges

File: openshift/e2e-common.sh [hunk 1]
```diff
@@ -146,6 +146,10 @@ function run_e2e_new_tests() {
 
   go_test_e2e -timeout=100m ./test/e2e_new/... "${common_opts[@]}" || return $?
   go_test_e2e -timeout=100m ./test/e2e_new_channel/... "${common_opts[@]}" || return $?
+
+  oc patch knativekafka --type merge -n "${EVENTING_NAMESPACE}" knative-kafka --patch-file "${SCRIPT_DIR}/knative-kafka-update-consumergroup-template.yaml"
```
Comment (Cali0707) [inline openshift/e2e-common.sh:150]: @pierDipi @matzew I'm not 100% sure if this is the right way to do this, lmk if I should change this

File: openshift/e2e-common.sh [hunk 2]
```diff
@@ -146,6 +146,10 @@ function run_e2e_new_tests() {
...
+  go_test_e2e -timeout=15m ./test/e2e_new -run TestTriggerUsesConsumerGroupIDFromTemplate || return $?
```
Comment (pierDipi) [inline openshift/e2e-common.sh:151]:
```suggestion
  go_test_e2e -timeout=15m ./test/e2e_new -run TestTriggerUsesConsumerGroupIDFromTemplate "${common_opts[@]}" || return $?
```

**Comment by pierDipi** [comment]
@Cali0707 we need to make sure to clone `release-1.32` here https://github.com/openshift-knative/eventing-kafka-broker/blob/edde43303fd9124abdd9367aa76b238775780ea5/openshift/e2e-common.sh#L78 

See how we do it in eventing core https://github.com/openshift-knative/eventing/blob/5cd673f934fb56312e32dd4037f0aff263ce6dbd/openshift/e2e-common.sh#L94

**Comment by Cali0707** [comment]
>  we need to make sure to clone release-1.32 here 

@pierDipi this should hopefully be fixed with my newest commit!

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**